src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -2,8 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
-import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
```

Comment:
the changes in this file revert the quick and dirty PR from Friday that totally hid/removed the Direct Deposit feature and showed users an Alert re: DD not being available.